content/os/Linux-Desktop-Hardening.md

```diff
@@ -105,6 +105,8 @@ One caveat with Snap packages is that you only have control over the interfaces
 
 ### Firejail
 
+{{< youtube id="N-Mso2bSr3o">}}
```

Comment:
the full 1080p60 non-cropped version is here: https://divested.dev/videos/Firejail-20210101.1080p.m4v

Reply:
Ah yeah thanks, though I think the youtube embed is okay for now. I may look into embeding videos from other sites later.